build/build.sh

```diff
@@ -11,10 +11,10 @@ chmod +x "$BIN"
 rm dist/cli.js
 
 # esbuild-plugin
-./dist/civet --js < source/esbuild-plugin.civet > dist/esbuild-plugin.js
+./dist/civet --js -c source/esbuild-plugin.civet -o dist/.js
```

Comment:
Why not use the piping here?

Reply:
The piping still works, but I figured that I built the CLI to make this more convenient, so why not use it? (It serves as an example of how to use the CLI.)

Reply:
I'd probably go with the long names or the `<` `>` in build scripts with a preference towards `< >` because it is an ancient and venerable bash tradition :)

Reply:
One more argument for not using `<`/`>`: on all operating systems (not just those with `/dev/tty` links), Civet will correctly know the input filename, in case there are bugs/errors during compilation.

I've switched to full filenames given your preference. I can switch fully back to `<`/`>` if you don't buy the argument above.